Functii/src/a/AverageCalculation.java

```diff
@@ -0,0 +1,35 @@
+package a;
+
+import java.util.Scanner;
+
+public class AverageCalculation {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("What is the length of the array?");
+        int n=sc.nextInt();
+        int[] m= arrayLength(n);
+        calculateAverage(m,n);
+    }
+    public static int[] arrayLength(int a){
+        Scanner scanner=new Scanner(System.in);
+        int [] array= new int [a];
+        for (int i=0;i<a;i++){
+            System.out.println("Please introduce the elements of the array["+ i +"] :");
+            array[i]= scanner.nextInt();
+        }
+        return array;
+
+    }
+
+    public static int calculateAverage(int[] array, int n){
+        int sum=0;
+        int average;
```

Comment:
Variabila in care vei stoca valoarea mediei este de tip `int`, deci media calculata de tine nu va avea zecimale.